controls/treeview/control-template.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Control Template
+page_title: .NET MAUI TreeView Documentation - Control Template
+description: Review what is the default control template of the Telerik .NET MAUI TreeView control and all elements we have in the template. 
```

Comment:
* Do you mean that there is a non-default Control Template? You don't mention anything about this in the article and you should.
* Instead of writing "all elements", list several of them.
* Don't use "we".